### PR TITLE
feat(security): security section for website

### PR DESCRIPTION
**Pull request by bpowell**

### CODE REVIEW

**Comment by bpowell** [comment]
@robzienert do you know if the security@spinnaker.io email has been setup?

File: community/security/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+## Responsible Disclosure Policy
+If you discover a security vulnerability, the Spinnaker Security SIG would like to know about it. The quicker we know, the quicker we can take steps to address the issue. We ask that you not publicly disclose the vulnerability until we have had the chance to investigate and determine the impact.
+
+How to report a security vulnerability in Spinnaker?
```
Comment (dorbin) [inline community/security/index.md]:
```suggestion
## How to report a security vulnerability in Spinnaker?

```

File: community/security/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+If you believe that you have found a security vulnerability in Spinnaker, please email security@spinnaker.io to report the issue. This email is only for reporting security vulnerabilities. For general help or bug reports please send those to either [Spinnaker Slack](https://join.spinnaker.io/) or [GitHub issues](https://github.com/spinnaker/spinnaker/issues) for the Spinnaker project. Include the following in the report:
```
Comment (dorbin) [inline community/security/index.md]:
```suggestion
If you believe that you have found a security vulnerability in Spinnaker, please email security@spinnaker.io to report the issue. Include the following in the report:
```

File: community/security/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+1. Description of the vulnerability
+1. Potential impact of the vulnerability
+1. A detailed description of how to reproduce the vulnerability (including scripts, screenshots, etc)
+
```
Comment (dorbin) [inline community/security/index.md:19]:
```suggestion
This email is only for reporting security vulnerabilities. For general help or bug reports please send those to either [Spinnaker Slack](https://join.spinnaker.io/) or [GitHub issues](https://github.com/spinnaker/spinnaker/issues). 

```

**Review by dorbin — COMMENTED** [review]
@bpowell 
Just a few suggestions...

File: community/security/index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+layout: single
+title:  "Security"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+## Responsible Disclosure Policy
+If you discover a security vulnerability, the Spinnaker Security SIG would like to know about it. The quicker we know, the quicker we can take steps to address the issue. We ask that you not publicly disclose the vulnerability until we have had the chance to investigate and determine the impact.
```
Comment (robzienert) [inline community/security/index.md]: Make the `Spinnaker Security SIG` linkable to the SIG page; or better yet directly to the SIG information table for Security.

File: community/security/index.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+## How to report a security vulnerability in Spinnaker?
+If you believe that you have found a security vulnerability in Spinnaker, please email security@spinnaker.io to report the issue. Include the following in the report:
```
Comment (robzienert) [inline community/security/index.md]: make `security@spinnaker.io` linkable.

**Comment by bpowell** [comment]
@dorbin @robzienert can this be merged please?

**Review by dorbin — APPROVED** [review]
LGTM

@robzienert , what say ye?

**Review by robzienert — APPROVED** [review]
Yar

**Pull request merged**